dali_tf_plugin/dali_dataset_op.cc

```diff
@@ -516,6 +516,22 @@ class DALIDatasetOp::Dataset::Iterator : public DatasetIterator<Dataset> {
     for (int out_id = 0; out_id < num_outputs; ++out_id) {
       TensorShape output_shape;
 
+      if (!daliIsUniformShapeAt(&pipeline_handle_, out_id)) {
```

Comment:
Maybe we should add this check to the old op as well?

Reply:
I can try to transplant it there.